docs/platforms/php/guides/laravel/index.mdx

```diff
@@ -105,13 +105,13 @@ Performance data is transmitted using a new event type called "transactions", wh
 
 ## Local Development and Testing
 
-When Sentry is installed in your application, it will also be active when you are developing or running tests.
+When Sentry is installed in your application, it will also be active when you are developing or running tests. Because PHP is single-threaded, Sentry will send events synchronously, which can slow down your application or test suite.
 
-You most likely don't want errors to be sent to Sentry when you are developing or running tests. To avoid this, set the DSN value to `null` to disable sending errors to Sentry.
+You most likely don't want errors to be sent to Sentry when you are developing or running tests so they don't use up [quota](/pricing/quotas/). To avoid this, set the DSN value to `null` to disable sending errors to Sentry.
```

Comment:
```suggestion
You most likely don't want errors to be sent to Sentry when you are developing or running tests so they don't use up [quota](/pricing/quotas/). To avoid this, set the DSN value to `null` to disable sending any events, check-ins, profiles or transactions to Sentry.
```

Reply:
And below :)